aries-site/src/pages/components/tooltip.mdx

```diff
@@ -0,0 +1,56 @@
+import { Example } from '../../layouts';
+import { 
+  ToolTipExitExample,
+  ToolTipIconExample,
+  ToolTipSimpleExample,
+} from '../../examples';
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tooltip/ToolTipSimpleExample.js"
+  docs="https://v2.grommet.io/tip?theme=hpe#props"
+  figma="https://www.figma.com/file/Jd34cI5yyZIPb5BIwkIPU2/HPE-Tooltip?node-id=0%3A1"
+>
+  <ToolTipSimpleExample />
+</Example>
+
+## Guidance
+
+Tooltips will display information upon hover and focus.
+Tooltips provide messages that give the user more information about an element.
+
+### Usage
+
+1. Don't use Tooltips for information that is vital to task completion.
+2. Provide a brief and helpful content inside the Tooltip.
+3. Support both mouse and keyboard hover.
+4. Use Tooltip arrows when multiple elements are nearby.
```

Comment:
I don't think we use arrows... probably should reword.

Reply:
Ill fix that